Oliver-Akins/CMPT-306-Group-Game
# Between levels UI - Make skills transfer between scenes - Also music

Testing: 
- start from the main menu scene
- load into the main game
- collect any number of coins before dropping down a hole to trigger between levels scene (you may have to drop the prefab into the scene)
- change stats
- leave between levels UI
- check stats to confirm changed (can just re-access between levels UI via hole)
